lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,522 @@
+
+      expect(dropdownElement).toHaveAttribute('aria-hidden', 'true');
+      inputElement.focus();
+      await userEvent.keyboard('[ArrowUp]');
+      expect(dropdownElement).toHaveAttribute('aria-hidden', 'false');
+      expect(lastOptBtn).toHaveAttribute('aria-selected', 'true');
+    });
+
+    it('should move visual focus to the previous option if dropdown is open' +
+      'and the up arrow key is pressed', async function() {
+      const { getByRole, getAllByRole } = quickRender({
+            matches: [{ id: '1', displayName: 'Foo' }, { id: '2', displayName: 'Boo' }] }),
+          inputElement = getByRole('combobox'),
+          firstOptBtn = getAllByRole('option', { hidden: true })[0],
+          secondOptBtn = getAllByRole('option', { hidden: true })[1];
+
+      inputElement.focus();
+      await userEvent.keyboard('[ArrowUp]');
+      expect(secondOptBtn).toHaveAttribute('aria-selected', 'true');
+      await userEvent.keyboard('[ArrowUp]');
+      expect(secondOptBtn).toHaveAttribute('aria-selected', 'false');
+      expect(firstOptBtn).toHaveAttribute('aria-selected', 'true');
+    });
+
+    it('should move visual focus to the last option if current focus is on the first option' +
+      'and the up arrow key is pressed', async function() {
+      const { getByRole, getAllByRole } = quickRender({
+            matches: [{ id: '1', displayName: 'Foo' }, { id: '2', displayName: 'Boo' }] }),
+          inputElement = getByRole('combobox'),
+          firstOptBtn = getAllByRole('option', { hidden: true })[0],
+          secondOptBtn = getAllByRole('option', { hidden: true })[1];
+
+      inputElement.focus();
+      await userEvent.keyboard('[ArrowDown]');
+      expect(firstOptBtn).toHaveAttribute('aria-selected', 'true');
+      await userEvent.keyboard('[ArrowUp]');
+      expect(firstOptBtn).toHaveAttribute('aria-selected', 'false');
+      expect(secondOptBtn).toHaveAttribute('aria-selected', 'true');
+    });
+
+    it('should close dropdown when Escape key is pressed', async function() {
+      const { getByRole } = quickRender({
+            matches: [{ id: '1', displayName: 'Foo' }, { id: '2', displayName: 'Boo' }] }),
+          inputElement = getByRole('combobox'),
+          dropdownElement = getByRole('listbox', { hidden: true });
+
+      inputElement.focus();
+      await userEvent.keyboard('[ArrowDown]');
+      expect(dropdownElement).toHaveAttribute('aria-hidden', 'false');
+      await userEvent.keyboard('[Escape]');
+      expect(dropdownElement).toHaveAttribute('aria-hidden', 'true');
+    });
+
+    it('sets the value to be backpaced text when backspace is pressed', async function() {
+      const onChange = jest.fn(),
+          { getByRole } = quickRender({ value: 'foo', onChange }),
+          inputElement = getByRole('combobox') as HTMLInputElement;
+
+      inputElement.focus();
+      inputElement.setSelectionRange(1, 2);
+      await userEvent.keyboard('[Delete]');
```

Comment:
Delete and Backspace are two different things are they not? The name of this test says it's supposed to be about Backspace

Reply:
It should be Backspace, updated here 6dbc601488edec3acdf06ca1c289688de300a1a8.